runtime/dali/src/lib.rs

```diff
@@ -910,6 +910,7 @@ impl mosaic::Config for Runtime {
 	type MinimumTimeLockPeriod = MinimumTimeLockPeriod;
 	type BudgetPenaltyDecayer = mosaic::BudgetPenaltyDecayer<Balance, BlockNumber>;
 	type NetworkId = u32;
+	type RemoteAssetId = [u8; 20];
```

Comment:
Can we make this an enum with 1 variant for now, as we will be storing these, we may want to add others later on. Enums will be easier to migrate storage wise.